verdict/src/main.rs

```diff
@@ -0,0 +1,123 @@
+use clap::{App, Arg};
+use serde_json::Value;
+use std::fs::File;
+use std::io::prelude::*;
+
+use configureProvider::configure_provider_service_client::ConfigureProviderServiceClient;
+use configureProvider::{ExportPolicyRequest, ExportPolicyResponse};
+use configureProvider::{SetPolicyRequest, SetPolicyResponse};
+use configureProvider::{SetRawPolicyRequest, SetRawPolicyResponse};
+
+pub mod configureProvider {
+    tonic::include_proto!("configureprovider");
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let matches = App::new("verdict")
+        .version("0.1")
+        .author("Inclavare-Containers Team")
+        .arg(
+            Arg::with_name("set_policy")
+                .short("s")
+                .long("set_policy")
+                .value_name("POLICY_NAME")
+                .value_name("FILE_REFERENCE")
+                .help("Generate a policy file named <POLICY_NAME>, according to the contents in <FILE_REFERENCE>.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("set_raw_policy")
+                .short("r")
+                .long("set_raw_policy")
+                .value_name("POLICY_NAME")
+                .value_name("FILE")
+                .help("Write the contents of <FILE> into the policy file named <POLICY_NAME>.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("export_policy")
+                .short("e")
+                .long("export_policy")
+                .value_name("POLICY_NAME")
```

Comment:
Maybe also need to add a "PATH" option which will designate where the exported policy need to store. And the "PATH" option is optional, by default the policy will store in the same folder of verdict's execution folder.